builtin/mainmenu/dlg_config_world.lua

```diff
@@ -33,14 +33,26 @@ local function get_formspec(data)
 
 	local hard_deps, soft_deps = pkgmgr.get_dependencies(mod.path)
 
+	if mod.is_modpack then
+		local info = minetest.formspec_escape(
+			pkgmgr.get_modpack_description(mod.path))
```

Comment:
Please may you use   core.get_content_info(), and get the description from    info.description

Reply:
done